test/test_helper.rb

```diff
@@ -2,12 +2,16 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require 'webmock'
+require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.